### DISCUSSION TITLE
Hover text for settings

### DISCUSSION THREAD

**Discussion by Iritscen** [Aleph-One-Marathon/alephone#259] [Ideas]
Many settings in AlephOne are not self-explanatory, as seen with my recent misunderstanding of what "Interface Button Sounds" means (sounds on the F-keys in-game, really?!).  Other examples of unclear settings would be "3D Models" and "3D Perspective", but I could go on and on.  Hover text that appears when mousing over a setting can provide useful elaboration on what each setting does.

**Comment by treellama** [comment]
We could definitely use better documentation of the settings (the wiki here would be a good place for that). However, hover text is not a good UI choice: it won't work for keyboard, controller, or touchscreen users.
